core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/ByteOrderedTokenFactory.java

```diff
@@ -16,6 +16,7 @@
 package com.datastax.oss.driver.internal.core.metadata.token;
 
 import com.datastax.oss.driver.api.core.metadata.token.Token;
+import com.datastax.oss.driver.api.core.metadata.token.TokenFactory;
```

Comment:
Why is this needed when you haven't changed this class?

Reply:
The whole story:

1. I added `TokenFactory getPartitioner()` to `com.datastax.oss.driver.api.core.cql.PreparedStatement` (`api` NOT `internal`).
2. After that, automatic checker in build system complained that `TokenFactory` is from `com.datastax.oss.driver.internal.core.metadata.token` (`internal`!). It complained that `api` cannot import stuff from `internal` (probably for automatic docs purposes).
3. That forced me to move `TokenFactory` from `internal` to `api`:
![image](https://user-images.githubusercontent.com/10003183/97442604-5298d280-192a-11eb-8875-86d8cb378793.png)
4. Other classes in `com.datastax.oss.driver.internal.core.metadata.token` used `TokenFactory`. Because before the change it was in the same package, no import was needed. After moving `TokenFactory` from `internal` to `api`, this import is needed.

Reply:
Ok makes sense but why isn't it a separate commit which describes what's going on and why?

Reply:
Separated into a52dc57efcd1ff7e1026ec0578081b8e175c9a1c.